pkgs/development/compilers/chicken/5/overrides.nix

```diff
@@ -271,6 +282,7 @@ in
   sundials = broken;
   svn-client = broken;
   tokyocabinet = broken;
+  vandusen = broken;
```

Comment:
This breakage was also fixed with https://github.com/NixOS/nixpkgs/pull/433689 so no need to add it anymore.